src/makeCalculator.js

```diff
@@ -37,7 +37,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(func, num) {
+      this.result = func(this.result, num);
+
+      return this;
+    },
+    add: (a, b) => a + b,
```

Comment:
The 'add', 'subtract', 'multiply' and 'divide' methods are currently implemented as arrow functions. This means they don't have their own 'this' context, which may lead to unexpected behaviour if they are used within the 'operate' method. Consider changing them to regular functions.